src/NServiceBus.Core/Transports/MessageContext.cs

```diff
@@ -68,6 +68,12 @@ public MessageContext(string messageId, Dictionary<string, string> headers, byte
         /// <summary>
         /// Context provided by the transport.
         /// </summary>
-        public ContextBag Context { get; }
+        [ObsoleteEx(Message = "Usage of context is discouraged.", ReplacementTypeOrMember = "Extensions", RemoveInVersion = "7")]
```

Comment:
How about "Usage of 'Context' is discouraged." instead? 

Alternatively, does that message really gain us anything over just having the obsolete there anyway?